packages/CollabCodeEdit-Tests.package/CCESessionTest.class/instance/testAddCollaboratorFromPayloadAtOtherJoinedSession.st

```diff
@@ -0,0 +1,11 @@
+tests collaborator managment
+testAddCollaboratorFromPayloadAtOtherJoinedSession
+	| expectedCollaborator | 
+	
```

Comment:
There is a lot of code duplication to the prior test here. Would be nice to have a dedicated method for that.